File: Codeforce problems/ab.py
t = int(input())

for i in range(0,t,1):
    count = 1
    a,b = list(map(int,input().split()))

    if(a==b): print(0)
    else:
        if(a>b): x = a/b
        else: x = b/a

        while True:
            count += 1
            if(x==8 or x==4 or x==2):
                print(count-1)
                break

            if(x%8 == 0): 
                x = x/8

            if(x==1 or x==3 or x==5 or x==6 or x==7 or x==9):
                print(-1)
                break
                    

File: Codeforces/ab.cpp
#include <bits/stdc++.h>
using namespace std;

int main()
{
    #ifndef ONLINE_JUDGE
        freopen("input.txt","r",stdin);
        freopen("output.txt","w",stdout);
    #endif


    long long unsigned int a,b,t,x,i,count,flag=0;
    cin >> t;

    while(t--)
    {
        
        flag = 0;
        cin >> a >> b;

        if(a<b) swap(a,b);
        if(a%b != 0 ) cout << -1 << endl;
        
        else
        {
            count = 0;
            x = a/b;
            
            while(x%8 == 0) x/=8,count++;
            while(x%4 == 0) x/=4,count++;
            while(x%2 == 0) x/=2,count++;

            if(x != 1) cout << -1 << endl;
            else cout << count << endl;
        }
       
    }

}